include/config.h

```diff
@@ -18,6 +18,12 @@ enum mako_button_binding {
 	MAKO_BUTTON_BINDING_INVOKE_DEFAULT_ACTION,
 };
 
+enum mako_sort_direction {
+	MAKO_SORT_DIRECTION_TIME_ASC = 1,
+	MAKO_SORT_DIRECTION_URGENCY_ASC = 2,
+	MAKO_SORT_DIRECTION_URGENCY = 4,
```

Comment:
Hmm, so I guess this is where there are some limitations to using a bitfield for this.

Maybe we could have a `sort` bitfield, which specifies which criteria is sorted, and another `sort_asc` bitfield, which specifies which criteria are sorted in ascending order (the default being descending). Thoughts?

Reply:
>Maybe we could have a sort bitfield, which specifies which criteria is sorted, and another sort_asc bitfield, which specifies which criteria are sorted in ascending order (the default being descending). Thoughts?